.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Run Unit Tests
+
+# Remove this for now, circleci is getting in the way with tests failing
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  # trigger on pull requests
+  pull_request:
+
+  # trigger on all commits to master and tagged commits
+  push:
+    branches:
+      - "master"
+    tags:
+      - "v*"
```

Comment:
```suggestion
```
If you aren't uploading wheels in this workflow, there is no need to run it on tags.